invertedai/api/drive.py

```diff
@@ -149,6 +150,9 @@ def drive(
         )
         return response
 
```

Comment:
You can add `warnings.simplefilter("once")` so it won't keep printing the same message over and over

Reply:
I tested this but I don't think it quite works with our stateless approach in drive.

Reply:
I made it work by applying the filter into the __init__.py file.